hatch_build.py

```diff
@@ -0,0 +1,14 @@
+from pathlib import Path
+
+from hatchling.builders.hooks.plugin.interface import BuildHookInterface
+
+
+class CustomBuildHook(BuildHookInterface):
+    def initialize(self, version, build_data):
+        from babel.messages.frontend import compile_catalog
+
+        cmd = compile_catalog()
+        cmd.directory = Path(__file__).parent / "ihatemoney" / "translations"
+        cmd.statistics = True
+        cmd.finalize_options()
+        cmd.run()
```

Comment:
Nitpick: could this be factorized somehow with what's done in the tests?

Reply:
Here it is. Note that I needed to put the babel related code in a dedicated file, because at build time no other package is installed yet. If `compile_catalogs` was in `utils.py` then package build would have fail on imports.